manifests/r/Roblox/RobloxStudio/1.0/Roblox.RobloxStudio.locale.en-US.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Roblox.RobloxStudio
+PackageVersion: 1.0
+PackageLocale: en-US
+Publisher: Roblox | Published to database by Markandpreston

```

```suggestion
Publisher: Roblox Corporation
```
to match Control Panel.

---

alright